.env.sample

```diff
@@ -2,6 +2,17 @@
 # HTTPS is assumed for security reasons
 DAEMON_URL=
 
+# These are for powering the LBRY Developer Program
+# /developer-program
+GITHUB_APP_ID=
+GITHUB_APP_SECRET=
+
+GITHUB_APP_ID_TEST=
```

Comment:
There should only be one set of values here, not two differently named ones. The production server receives the production values; local development can use the test values.